apps/src/sites/studio/pages/levels/show.js

```diff
@@ -61,14 +61,21 @@ function initPage() {
 
   if (hasScriptData('script[data-aitutordata]')) {
     const aiTutorData = getScriptData('aitutordata');
-    const {levelId, type, hasValidation, isAssessment, isProjectBacked} =
-      aiTutorData;
+    const {
+      levelId,
+      type,
+      hasValidation,
+      isProjectBacked,
+      aiTutorAvailable,
+      isAssessment,
+    } = aiTutorData;
     const level = {
       id: levelId,
       type: type,
       hasValidation: hasValidation,
-      isAssessment: isAssessment,
       isProjectBacked: isProjectBacked,
+      aiTutorAvailable,
+      isAssessment: isAssessment,
```

Comment:
I think you're going to have a merge conflict with yourself with https://github.com/code-dot-org/code-dot-org/pull/57051. Let me know if you want extra eyes resolving. Same comment here: let's be consistent with the fields.